### PR TITLE
clone-remote-port option

### PR DESCRIPTION
**Pull request by blackalegator**
Added a clone-remote-port option which makes the bedrock's port the same as the java's port if geyser is running as a plugin

### CODE REVIEW

File: connector/src/main/resources/config.yml [hunk 1]
```diff
@@ -13,6 +13,9 @@ bedrock:
   address: 0.0.0.0
   # The port that will listen for connections
   port: 19132
+  # Clone remote server port number
+  # Geyser will try to set bedrock port to the same as java's server port if it is run as a plugin
```
Comment (DoctorMacc) [inline connector/src/main/resources/config.yml:17]: I would change this language to something like:
```
# Use Java server port
# This option ignores the above port and uses the Java server's port as Bedrock's port. Only available for plugin versions.

Comment (blackalegator) [reply connector/src/main/resources/config.yml]: I've made the edit, copied uwu's desc.

Comment (E404NNF) [reply connector/src/main/resources/config.yml]: OwO?! Someone said my name

**Comment by blackalegator** [comment]
> I would also add a check for this on Sponge's end too, just for consistency.

Done

**Comment by DoctorMacc** [comment]
Please update this PR to support 1.16.

**Comment by RednedEpic** [comment]
Looks like you have a merge conflict that needs to be resolved. Once that is taken care of, I can merge this 🙂

**Comment by DoctorMacc** [comment]
Superseeded by #944.

**Comment by RednedEpic** [comment]
Reopening since #944 does not include this commit history. Closed before realizing this was the case; sorry for the confusion!

**Comment by RednedEpic** [comment]
Due to the lack of response from OP, this has been superseded by #1061. Commit history from this PR has been kept however. Regardless, thanks for the contribution!

**Pull request closed without merging**